gdx/src/com/badlogic/gdx/math/Intersector.java

```diff
@@ -1049,158 +1049,121 @@ public static boolean overlapConvexPolygons (float[] verts1, float[] verts2, Min
 	}
 
 	/** Check whether polygons defined by the given counter-clockwise wound vertex arrays overlap. If they do, optionally obtain a
-	 * Minimum Translation Vector indicating the minimum magnitude vector required to push the polygon defined by verts1 out of the
-	 * collision with the polygon defined by verts2.
-	 * @param verts1 Vertices of the first polygon.
-	 * @param verts2 Vertices of the second polygon.
+	 * Minimum Translation Vector indicating the minimum magnitude vector required to push the polygon defined by shapeA out of the
+	 * collision with the polygon defined by shapeB.
+	 * @param shapeA Vertices of the first polygon.
+	 * @param shapeB Vertices of the second polygon.
```

Comment:
I don't particularly like renaming `verts` to `shape` but if we do, we should rename the other `overlapConvexPolygon` method(s) that receive vertices as an argument to follow the same naming.

Reply:
I'd prefer we leave the original naming.

Reply:
Yes i will change it to the original naming scheme.